src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -161,9 +159,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     return Optional.ofNullable(results.asSingleEntity());
   }
 
-  @VisibleForTesting
   /** Returns YouTube video ID for a given {@code videoUrl}. */
-  protected Optional<String> getVideoId(String videoUrl) {
+  // TODO: Move this function to a Utils class.
```

Comment:
It is related to my PR because I need the video id to get the transcript language options. I'll remove the TODO though because I probably won't have time to do it.

Reply:
Please keep TODOs even if you don't get to them because it is important information.

Reply:
gotcha, re-added.